### ISSUE TITLE
Pagination doesn't work correctly ?

### ISSUE THREAD

**Issue by ciadch** [Adldap2/Adldap2#55]
Hi,

I have an error when I try to user pagination : 

```
Catchable fatal error: Argument 1 passed to Adldap\Objects\Paginator::__construct() must be of the type array, null given
```

And here is my function :
```php
$ad->users()->search()->paginate(50, 0);
```

Thanks

**Comment by stevebauman** [comment]
Apologies, this was due to the collection changes. Released `v5.1.3` that fixes this. Working on a ton more tests atm.

**Comment by ciadch** [comment]
Perfect thanks !
